data/strings/types_strings.txt

```diff
@@ -15752,55 +15828,63 @@
 
   [type.railway.razed]
     en = Razed Rail
+    de = Stillgelegte Eisenbahnstrecke
     ja = 撤去済の鉄道
     pt = Ferrovia removida
     pt-BR = Ferrovia removida
     ru = Заброшенная Ж/Д
 
   [type.railway.siding]
     en = Siding Rail
+    de = Ausweich-/Überholgleis
     ja = 側線
     pt = Desvio de ferrovia
     pt-BR = Desvio de ferrovia
     ru = Ветка Ж/Д
 
   [type.railway.siding.bridge]
     en = Siding Rail
+    de = Ausweich-/Überholgleis
     ja = 側線(橋)
     pt = Desvio de ferrovia
     pt-BR = Desvio de ferrovia
     ru = Ветка Ж/Д
 
   [type.railway.siding.tunnel]
     en = Siding Rail
+    de = Ausweich-/Überholgleis
     ja = 側線(トンネル)
     pt = Desvio de ferrovia
     pt-BR = Desvio de ferrovia
     ru = Ветка Ж/Д
 
   [type.railway.spur]
     en = Spur Rail
+    de = Anschlussgleis
     ja = 引き込み線
     pt = Ramal de ferrovia
     pt-BR = Ramal de ferrovia
     ru = Подъездная Ж/Д
 
   [type.railway.spur.bridge]
     en = Spur Rail
+    de = Anschlussgleis
     ja = 引き込み線(橋)
     pt = Ramal de ferrovia
     pt-BR = Ramal de ferrovia
     ru = Подъездная Ж/Д
 
   [type.railway.spur.tunnel]
     en = Spur Rail
+    de = Anschlussgleis
     ja = 引き込み線(トンネル)
     pt = Ramal de ferrovia
     pt-BR = Ramal de ferrovia
     ru = Подъездная Ж/Д
 
   [type.railway.station]
     en = Train Station
+    de = Bahnhof
```

Comment:
de translation is already present below. Please re-check to avoid duplicates.

Reply:
done, I removed 4 duplicates and the accidental shift and pushed my changes.